### ISSUE TITLE
pod install command not worked with React Native v0.69.0

### ISSUE THREAD

**Issue by YangJonghun** [react-native-google-cast/react-native-google-cast#438]
current locate of `react-native-google-cast.podspec` is not worked properly with React Native v0.69.0.
every Pod files related react-native-google-cast doesn't install with `$ pod install --project-directory=ios` command

so I temporarily change path indicated below. and it works
- as-is
  - node_modules/react-native-google-cast/ios/react-native-google-cast.podspec
- to-be
  - node_modules/react-native-google-cast/react-native-google-cast.podspec
```diff
 require 'json'
-package = JSON.parse(File.read(File.join(__dir__, '..', 'package.json')))
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
 
 Pod::Spec.new do |s|
   s.name     = package['name']
```

I think root cause is changing of React Native v0.69.0's react native cli version (v8)
(FYI. https://reactnative.dev/blog/2022/06/21/version-069)
(FYI2. https://github.com/react-native-community/cli/pull/1537)

**Comment by YangJonghun** [comment]
@petrbela 
v4.2.5 build files doesn't include any podspec file, please check current build config

**Comment by petrbela** [comment]
My bad. Just released 4.2.6, let me know if it still doesn't work.
